LabApiUtilities/src/test/com/microsoft/identity/labapi/utilities/client/LabGuestAccountTest.java

```diff
@@ -0,0 +1,28 @@
+package com.microsoft.identity.labapi.utilities.client;
```

Comment:
Missing LICENSE

Reply:
I hope one of these days I'll create a PR without forgetting to add all the licenses